multi-project/core/src/main/java/by/artempvn/lab01/core/Utils.java

```diff
@@ -0,0 +1,20 @@
+package by.artempvn.lab01.core;
+
+import by.artempvn.lab01.util.StringUtils;
+
+public class Utils {
+
+	public static boolean isAllPositiveNumbers(String... str) {
+		if (str == null) {
+			return false;
+		}
```

Comment:
It would be better to avoid null-checks. In case of `str` is null for some reason - this method should not have been called at all.

Reply:
So in future I do not need any checks for null-value of argyment?

Reply:
Correct. It is a bad practice to use `null` in the code. Better way - is to use Optional API or throw an Exception. As a result, if some value is `null` - it is an exceptional situation and exception should be thrown as soon as possible in order to better indicate root issue.
In case of all parameters in code are optional - it will work, but with a huge amount of bugs.